packages/web/components/view-mode.tsx

```
@@ -0,0 +1,41 @@
+}
+
+export function ViewMode({ note, onToggleViewMode, onExport }: ViewModeProps) {
+  return (
+    <div className='p-4'>
+      <div className='flex items-center justify-between mb-4'>
+        <h1 className='text-2xl font-bold'>{note.title || 'Untitled'}</h1>
+        <div className='flex gap-2'>
+          <button
+            onClick={onToggleViewMode}
+            className='px-3 py-1 text-sm border rounded'
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)